src/components/SkyStateGenerator.cpp

```diff
@@ -183,12 +183,12 @@ const std::shared_ptr<SkyState> SkyStateGenerator::update(float dayRatio)
   return mCurrentSkyState;
 }
 
-std::tuple<unsigned int, unsigned int> SkyStateGenerator::findPresetIndices() const
+std::tuple<std::uint32_t, std::uint32_t> SkyStateGenerator::findPresetIndices() const
```

Comment:
I would prefer the fixed size types provided by bsf to stay consistent actually, ie. `bs::UINT32`. Do you mind switching the code them?

Reply:
Of course not, will do. I personally just prefer to use `std` stuff, and apart from that I think I remember saying someone that bsf are thinking about deprecating those typedefs anyway (I have no source for that tho)